meson.build

```diff
@@ -4,13 +4,18 @@ project(
   version: '1.19.1',
   meson_version: '>= 0.60',
   license: 'LGPL-2.0-or-later',
-  default_options: ['warning_level=2'])
+  default_options: [
+    'warning_level=2',
+    'force_fallback_for=libportal',
```

Comment:
Do we really want to force this? Seems fine to support both ways.

Reply:
With Debian hat on, I'd prefer to use the packaged libportal for the tests (I already do, and I'd like to continue to do that). If the released libportal doesn't have something that the x-d-p's test suite needs, the solution should be to release libportal.

Reply:
This isn't really forced as it changes the default. One can compile with `-Dforce_fallback_for=[]` to use the system provided libportal.

Reply:
@swick what you are suggesting is unnatural and non-idiomatic usage of meson. Nobody expects to have to do this.

Why do you even want to "change the default"? Surely changing a default should come with a rationale for why the default is bad.

Reply:
libportal is a dependency for the tests and the tests only. The tests are mainly a developer tool and getting the right version to developers as soon as possible is important. There are definitely packages which pass `-Dforce_fallback_for=[]` to meson already.

@eli-schwartz if you continue with this level of hostility I will block you

Reply:
We don't pass `force_fallback_for` for any of the packages using Meson in Gentoo. This is the first time it's come up.

Can you explain what the actual need here is? Why not just set the minimum version appropriately?

Reply:
Got rid of `force_fallback_for=[]` from the default_options but still using it in the CI. It now also has a version requirement.